bin/i18n/translation_monitor/configs/crowdin_validator_config.json

```diff
@@ -6,15 +6,226 @@
     "start_date": "2021-11-01"
   },
   "data": [
+    {
+      "project_name": "codeorg",
+      "crowdin_language_id": "ar",
```

Comment:
Seems like it would be nice if we could list multiple `crowdin_language_id`s in a single configuration